tf2onnx/graph.py

```diff
@@ -418,6 +418,11 @@ def make_const(self, name, np_val, skip_conversion=False, raw=True):
     def make_node(self, op_type, inputs, attr=None, output_count=1, outputs=None, skip_conversion=True,
                   op_name_scope=None, name=None, shapes=None, dtypes=None):
         """Make a new onnx node in the graph"""
+        utils.make_sure(isinstance(inputs, list) and (not inputs or isinstance(inputs[0], str)),
+                        "inputs should be a list of string")
+        utils.make_sure(outputs is None or (isinstance(outputs, list) and isinstance(outputs[0], str)),
+                        "outputs should be a list of string")
```

Comment:
not feel good about ```isinstance(inputs[0], str```, let's find a better way.

Reply:
delete them